contrib/mesos/pkg/queue/historical/historical.go

```diff
@@ -106,7 +129,7 @@ func (f *HistoricalFIFO) Add(v interface{}) error {
 	if entry, exists := f.items[id]; !exists {
 		f.queue = append(f.queue, id)
 	} else {
-		if entry.Is(DELETE_EVENT | POP_EVENT) {
+		if entry.Is(queue.DELETE_EVENT | queue.POP_EVENT) {
```

Comment:
I left all the events in queue because Event way required for the FIFO interface. But I can move the constance to historical, if you prefer.